app/about_us/OurValues.js

```diff
@@ -30,7 +30,9 @@ const OurValues = () => {
         return (
           <li key={value.title} className={styles.valueItem}>
             <h3 className={styles.valueTitle}>
-              <span className={styles.valueNumber}>{`0${index + 1}`}</span>
+              <span className={styles.valueNumber}>
+                {String(index + 1).padStart(2, "0")}
```

Comment:
woooow fancy logic here, I like it 😄

Reply:
I didn't even know this `.padStart` string function existed 😂

Reply:
I came across this method by accident and decided to apply it so as not to forget 😄